src/ossl/aes.rs

```diff
@@ -1967,6 +2057,11 @@ impl Decryption for AesOperation {
         Ok(outlen)
     }
 
+    /// Provides the expect buffer size for the provided input ciphertext length
```

Comment:
```suggestion
    /// Provides the expected decrypted output buffer size for the provided input ciphertext length
```
the same clarification should come to the message-based functions below